THANKS.txt

```diff
@@ -236,7 +236,9 @@ Sam Wallan for scipy.linalg.lapack enhancements
 Richard Weiss for a bug fix in scipy.optimize._differentialevolution.py.
 Luigi F. Cruz for adding time/frequency domain option to scipy.signal.resample.
 Wesley Alves for improvements to scipy.stats.jarque_bera and scipy.stats.shapiro
+<<<<<<< HEAD
```

Comment:
rebase error

Reply:
Aargh, don't know how I missed that, sorry.